README.md

```diff
@@ -160,6 +160,11 @@ which uses musl instead.
 
 [Alpine grpc-java package]: https://pkgs.alpinelinux.org/package/edge/testing/x86_64/grpc-java
 
+If you're running on a JDK version older than JDK 8, you'll also need to add the
+dependency `javax.annotation:javax.annotation-api`
+
+[javax.annotation.api]: https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api/1.3.2
```

Comment:
We are currently using v1.2 across the repository, because v1.3 does not support JDK 7.

Reply:
My bad there, I just included the latest version. I removed references to an explicit version, and just mentioned it had to be included if above JDK 8.